client/app/account/user/dashboard/user-dashboard.html

```diff
@@ -0,0 +1,74 @@
+<div class="module-useraccount-sections-dashboard">
+    <div class="tab-content">
+        <div data-ovh-alert="useraccount.alerts.dashboard"></div>
+
+        <div class="row pt-4">
+
+            <div class="col mb-4 mx-2">
+                <oui-tile data-heading="{{:: 'user_account_dashboard_shortcut' | translate }}">
+                    <oui-tile-button
+                        data-ng-repeat="shortcut in $ctrl.shortcuts track by $index"
+                        data-href="{{:: shortcut.href }}">
+                        <span data-translate="{{ 'user_account_dashboard_shortcut_' + shortcut.key | translate }}"></span>
+                    </oui-tile-button>
+                </oui-tile>
+            </div>
+
+            <div class="col mb-4 mx-2">
+                <oui-tile data-heading="{{:: 'user_account_dashboard_last_bill' | translate }}">
+                    <oui-tile-definition data-term="{{:: 'user_account_dashboard_last_bill_reference' | translate }}"
+                        data-description="{{:: $ctrl.lastBill.billId }}">
+                    </oui-tile-definition>
+                    <oui-tile-definition data-term="{{:: 'user_account_dashboard_last_bill_date' | translate }}"
+                        data-description="{{:: $ctrl.lastBill.date | date}}">
+                    </oui-tile-definition>
+                    <oui-tile-definition data-term="{{:: 'user_account_dashboard_last_bill_amount' | translate }}"
+                        data-description="{{:: $ctrl.lastBill.priceWithTax.text }}">
+                    </oui-tile-definition>
+                    <a
+                        class="oui-button oui-button_secondary mx-auto my-5 d-block"
+                        data-ng-href="{{ $ctrl.lastBill.url }}"
+                        target="_blank"
+                        rel="noopener"
+                        data-translate="user_account_dashboard_last_bill_see">
+                    </a>
+                </oui-tile>
+            </div>
+
+            <div class="col mb-4 mx-2">
+                <oui-tile data-heading="{{:: 'user_account_dashboard_profile' | translate }}">
+                    <div class="row">
+                        <div class="col-md-3">
+                            <span class="oui-icon oui-icon-user_extra-thin"></span>
+                        </div>
+                        <div class="col-md-7">
+                            <div class="mb-4">
+                                <p>
```

Comment:
Looks like
```suggestion
                                <strong class="d-block">
```
Would be more adapted (it's not really a paragraph, is it ?)